README.md

```diff
@@ -217,15 +221,18 @@ To simplify integration with TypeScript, two files - `typings/main.d.ts` and `ty
 }
 ```
 
-Or as a reference to the top of TypeScript files:
+* Add as a reference to the top of TypeScript files:
 
 ```ts
 /// <reference path="../typings/main.d.ts" />
 ```
 
-If you're building a front-end package it's recommended you use `typings/browser.d.ts` instead. The browser typings are compiled by following the `browser` field overrides.
-
-**Please note:** If you're relying on "exclude" behavior instead, you can exclude `typings/browser.d.ts` and `typings/browser` (replace `browser` with `main`, if you desire).
+* Exclude the one you don't want from `tsconfig.json` (similar to `node_modules`) e.g: 
+```json
+{
+  "exclude":["typings/browser.d.ts","typings/browser","node_modules"]
```

Comment:
My only gripe is the lack of whitespace here. Mind if we make each of these on a line?